x-pack/plugins/apm/public/components/shared/MetadataTable/MetadataTable.test.tsx

```diff
@@ -11,7 +11,7 @@ import { MemoryRouter } from 'react-router-dom';
 import { MetadataTable } from '.';
 import { MockApmPluginContextWrapper } from '../../../context/apm_plugin/mock_apm_plugin_context';
 import { expectTextsInDocument } from '../../../utils/testHelpers';
-import { SectionsWithRows } from './helper';
+import { SectionDescriptor } from './types';
```

Comment:
```suggestion
import type { SectionDescriptor } from './types';
```

it's just a test so doesn't matter for bundle size, but still nice to do.

Reply:
++, I think type imports are removed for jest tests as well, which helps with performance (with jest rewriting all the import statements)